org.flightgear.FlightGear.json

```diff
@@ -0,0 +1,193 @@
+{
+	"id":"org.flightgear.FlightGear",
+	"runtime":"org.kde.Platform",
+	"runtime-version":"5.10",
+	"sdk":"org.kde.Sdk",
+	"command":"flightgear.sh",
+	"rename-icon":"flightgear",
+	"finish-args":[
+		"--share=ipc",
+		"--socket=x11",
+		"--socket=wayland",
+		"--device=dri",
+		"--socket=pulseaudio",
+		"--share=network",
+		"--filesystem=~/.fgfs/"
```

Comment:
`--persist=.fgfs`

Reply:
This is needed so one can use additional aircraft. By default, there's only one shipped with the Flatpak.

Reply:
The directory is still accessible. It is a hidden directory anyway so it requires user documentation no matter what.